sdk/keyvault/Azure.Security.KeyVault.Administration/readme.md

```diff
@@ -1,34 +1,194 @@
+
+The Azure Key Vault administration library clients supports administrative tasks such as full backup / restore and key level role based access control (RBAC).
 
 ## Getting started
 
-Content forthcoming
+### Install the package
+Install the Azure Key Vault administration client library for .NET with [NuGet][nuget]:
 
-### Prerequisites
+```PowerShell
+Install-Package Azure.Security.KeyVault.Administration
+```
 
-Content forthcoming
+### Prerequisites
+* An [Azure subscription][azure_sub].
+* An existing Azure Key Vault. If you need to create an Azure Key Vault, you can use the Azure Portal or [Azure CLI][azure_cli].
 
-### Install the package
+If you use the Azure CLI, replace `<your-resource-group-name>` and `<your-key-vault-name>` with your own, unique names:
 
-Content forthcoming
+```PowerShell
+az keyvault create --resource-group <your-resource-group-name> --name <your-key-vault-name>
+```
 
 ### Authenticate the client
-
-Content forthcoming
+In order to interact with the Key Vault service, you'll need to create an instance of the [KeyClient][rbac_client] class. You need a **vault url**, which you may see as "DNS Name" in the portal,
+ and **client secret credentials (client id, client secret, tenant id)** to instantiate a client object.
+
+Client secret credential authentication is being used in this getting started section but you can find more ways to authenticate with [Azure identity][azure_identity]. To use the [DefaultAzureCredential][DefaultAzureCredential] provider shown below,
+or other credential providers provided with the Azure SDK, you should install the Azure.Identity package:
+
+```PowerShell
+Install-Package Azure.Identity
+```
+
+#### Create/Get credentials
+Use the [Azure CLI][azure_cli] snippet below to create/get client secret credentials.
+
+ * Create a service principal and configure its access to Azure resources:
+    ```PowerShell
+    az ad sp create-for-rbac -n <your-application-name> --skip-assignment
+    ```
+    Output:
+    ```json
+    {
+        "appId": "generated-app-ID",
+        "displayName": "dummy-app-name",
+        "name": "http://dummy-app-name",
+        "password": "random-password",
+        "tenant": "tenant-ID"
+    }
+    ```
+* Take note of the service principal objectId
+    ```PowerShell
+    az ad sp show --id <appId> --query objectId
+    ```
+    Output:
+    ```
+    "<your service principal's object id"
+    ```
+* Use the returned credentials above to set  **AZURE_CLIENT_ID** (appId), **AZURE_CLIENT_SECRET** (password), and **AZURE_TENANT_ID** (tenant) environment variables. The following example shows a way to do this in Powershell:
+    ```PowerShell
+    $Env:AZURE_CLIENT_ID="generated-app-ID"
+    $Env:AZURE_CLIENT_SECRET="random-password"
+    $Env:AZURE_TENANT_ID="tenant-ID"
+    ```
+
+* Grant the above mentioned application authorization to perform key operations on the Azure Key Vault:
```

Comment:
This should be tailored to what the administration client requires. It seems to be copied and pasted from the Keys README (great way to start a new README and keep it consistent!). Off hand, I imagine those probably include backup, restore, and list. We should find out / test to be sure.